__snapshots__/chart-yaml.js

```diff
@@ -0,0 +1,5 @@
+exports['Chart.yaml updateContent updates version in Chart.yaml 1'] = `
+version: 1.1.0
+
```

Comment:
these extra few newlines are a bit weird, perhaps confirm this is only in the snapshot, and not happening when the PR is created?

Reply:
fixed